torchtitan/utils.py

```diff
@@ -48,6 +48,10 @@ def set_determinism(seed: Optional[int]) -> None:
         torch.backends.cudnn.benchmark = False
         # set Python seed
         os.environ["PYTHONHASHSEED"] = str(seed)
+        torch.use_deterministic_algorithms(True)
+        # env var for deterministic CuBLAS
+        # https://github.com/pytorch/pytorch/blob/18525e185e211b3eab44c67a688e5df8396f6f97/torch/__init__.py#L1300
```

Comment:
can we cite https://pytorch.org/docs/stable/generated/torch.use_deterministic_algorithms.html
instead of the permanent link?